Sources/Guaka/CommandType/CommandType+Execution.swift

```diff
@@ -27,7 +46,29 @@ extension CommandType {
     return actualCommand(forCommand: self, args: Array(commandLineArgs.dropFirst())).0
   }
   
-  private func handleResult(_ result: Result) {
+}
+
+
+extension CommandType {
+  func findAdequateInheritableRun(forPre: Bool) -> Any?  {
+    
+    var cmd: CommandType? = self
+    
+    while true {
+      var toFind: Any?
```

Comment:
Do we need to use `Any` here? Don't we have something like the "Closure/Function" Type?